<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>983</width>
    <height>781</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item row="5" column="0">
       <widget class="QScrollBar" name="scrollbarCoronal">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QScrollBar" name="scrollbarAxial">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTipDuration">
         <number>-1</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <layout class="QGridLayout" name="gridCanvas3D">
        <property name="sizeConstraint">
         <enum>QLayout::SetMaximumSize</enum>
        </property>
       </layout>
      </item>
      <item row="4" column="0">
       <layout class="QGridLayout" name="gridAxial"/>
      </item>
      <item row="6" column="0">
       <layout class="QGridLayout" name="gridCoronal"/>
      </item>
      <item row="6" column="1">
       <layout class="QGridLayout" name="gridSagital"/>
      </item>
      <item row="5" column="1">
       <widget class="QScrollBar" name="scrollbarSagital">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="maximumSize">
       <size>
        <width>290</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>271</width>
          <height>71</height>
         </rect>
        </property>
        <property name="title">
         <string>Cargar Datos</string>
        </property>
        <widget class="QPushButton" name="pushNibabelabrir">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>91</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Abrir .nii</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushDicomabrir">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>20</y>
           <width>91</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Carpeta DICOM</string>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>47</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Fuente:</string>
         </property>
        </widget>
        <widget class="QLabel" name="labelRuta">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>50</y>
           <width>151</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>C:/....</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>80</y>
          <width>271</width>
          <height>141</height>
         </rect>
        </property>
        <property name="title">
         <string>Información</string>
        </property>
        <widget class="QScrollArea" name="scrollArea">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>251</width>
           <height>111</height>
          </rect>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>249</width>
            <height>109</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="2" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Axial ratio: </string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Coronal ratio:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_1">
             <property name="text">
              <string>Tamaño: </string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Pixdim: </string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="labelPixdim">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="labelShape">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Sagital ratio:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="labelAxialratio">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="labelCoronalratio">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLabel" name="labelSagitalratio">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupRDI">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>280</y>
          <width>281</width>
          <height>371</height>
         </rect>
        </property>
        <property name="title">
         <string>Region de Interes</string>
        </property>
        <widget class="QPushButton" name="pushCrea">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>82</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Crear Región</string>
         </property>
        </widget>
        <widget class="QGroupBox" name="groupPlandes">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>20</y>
           <width>161</width>
           <height>91</height>
          </rect>
         </property>
         <property name="title">
          <string>Plano de desplazamiento</string>
         </property>
         <widget class="QRadioButton" name="radioAxial">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>82</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Axial</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QRadioButton" name="radioCoronal">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>82</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Coronal</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="radioSagital">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>60</y>
            <width>82</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Sagital</string>
          </property>
         </widget>
        </widget>
        <widget class="QGroupBox" name="groupNave">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>120</y>
           <width>261</width>
           <height>241</height>
          </rect>
         </property>
         <property name="title">
          <string>Navegación</string>
         </property>
         <widget class="QWidget" name="gridLayoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>239</width>
            <height>141</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="1">
            <widget class="QPushButton" name="pushSubir">
             <property name="text">
              <string>↑</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="pushAceptar">
             <property name="text">
              <string>Aceptar</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="pushAgrandar">
             <property name="text">
              <string>Agrandar</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="pushAchicar">
             <property name="text">
              <string>Achicar</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="pushDerecha">
             <property name="text">
              <string>→</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Paso (pix):</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="pushIzquierda">
             <property name="text">
              <string>←</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="pushBajar">
             <property name="text">
              <string>↓</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLineEdit" name="linePaso">
             <property name="text">
              <string>1</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>210</y>
            <width>261</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>* Las caras axiales de la región deben ser cuadradas</string>
          </property>
         </widget>
         <widget class="QLabel" name="labelRDIaxial">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>20</y>
            <width>31</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_8">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>31</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Axial:</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_9">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>20</y>
            <width>41</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Coronal:</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_10">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>20</y>
            <width>41</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Sagital:</string>
          </property>
         </widget>
         <widget class="QLabel" name="labelRDIcoronal">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>20</y>
            <width>31</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
         <widget class="QLabel" name="labelRDIsagital">
          <property name="geometry">
           <rect>
            <x>210</x>
            <y>20</y>
            <width>31</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </widget>
        <widget class="QPushButton" name="pushReset">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>81</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>220</y>
          <width>271</width>
          <height>61</height>
         </rect>
        </property>
        <property name="title">
         <string>Coordenadas</string>
        </property>
        <widget class="QLabel" name="label_13">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>31</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Axial:</string>
         </property>
        </widget>
        <widget class="QLabel" name="labelAxial">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>20</y>
           <width>31</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_15">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>20</y>
           <width>41</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Coronal:</string>
         </property>
        </widget>
        <widget class="QLabel" name="labelCoronal">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>20</y>
           <width>31</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_17">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>20</y>
           <width>41</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Sagital:</string>
         </property>
        </widget>
        <widget class="QLabel" name="labelSagital">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>20</y>
           <width>31</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>6</x>
           <y>40</y>
           <width>61</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Punto RDI : </string>
         </property>
        </widget>
        <widget class="QLabel" name="labelRdipunto">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>40</y>
           <width>181</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>0,0,0,0,0</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>281</width>
          <height>141</height>
         </rect>
        </property>
        <property name="title">
         <string>Red Neuronal</string>
        </property>
        <widget class="QPushButton" name="pushButton">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>50</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Segmentar</string>
         </property>
        </widget>
        <widget class="QProgressBar" name="progressBar">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>110</y>
           <width>271</width>
           <height>23</height>
          </rect>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>983</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
